lib/Github/Api/PullRequest.php

```diff
@@ -92,6 +93,24 @@ public function files($username, $repository, $id)
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/files');
     }
 
+    /**
+     * All statuses which are the statuses of its head branch.
+     *
+     * @see http://developer.github.com/v3/pulls/
+     *
+     * @param string $username   the username
+     * @param string $repository the repository
+     * @param string $id         the ID of the pull request for which statuses are retrieved
+     *
+     * @return array array of statuses for the project
+     */
+    public function status($username, $repository, $id)
+    {
+        $link = $this->show($username, $repository, $id)['_links']['statuses']['href'];
```

Comment:
It would throw an http error same as it would if you called that method directly. It either passes and returns the given response or it fails and the whole chain blows up. According the the GitHub api those array keys are guaranteed